### PR TITLE
doc: Update maint process for k8s changes

### PR DESCRIPTION
**Pull request by ryanfaircloth**
Add metadata to chart.yaml and add maint docs instructions

### CODE REVIEW

**Comment by satoru-takeuchi** [comment]
We don't use `kubeVersion` to allow users to use topolvm with unsupported k8s versions at their own risk. It's intentional.

File: charts/topolvm/Chart.yaml [hunk 1]
```diff
@@ -6,6 +6,7 @@ name: topolvm
 description: Topolvm
 version: 13.0.1
 appVersion: 0.25.0
+kubeVersion: "1.25-1.27"
```
Comment (satoru-takeuchi) [inline charts/topolvm/Chart.yaml:9]: We don't use kubeVersion to allow users to use topolvm with unsupported k8s versions at their own risk. It's intentional.

File: docs/maintenance.md [hunk 2]
```diff
@@ -17,53 +17,56 @@ The format is up to you, but this is very important to keep track of what change
 Basically, we should pay attention to breaking changes and security fixes first.
 If we find some interesting features added in new versions, please consider if we are going to use them or not and make a GitHub issue to incorporate them after the upgrading task is done.
 
-Note: Ubuntu 18.04 is used to keep the backward compatibility of XFS formatting. (https://github.com/topolvm/topolvm/pull/306)
+Note: Ubuntu 18.04 is used to keep the backward compatibility of XFS formatting. (<https://github.com/topolvm/topolvm/pull/306>)
```
Comment (satoru-takeuchi) [inline docs/maintenance.md:20]: Is there any differences between with or without `<` and `>` for readers? At a first glance, these characters are not visible in the formatted markdown.

File: docs/maintenance.md [hunk 3]
```diff
@@ -17,53 +17,56 @@ The format is up to you, but this is very important to keep track of what change
...
+- `charts/topolvm/Chart.yaml` "kubeVersion"
 
 Next, we should update `go.mod` by the following commands.
 Please note that Kubernetes v1 corresponds with v0 for the release tags. For example, v1.17.2 corresponds with the `v0.17.2` tag.
+
 ```console
-$ VERSION=<upgrading Kubernetes release version>
-$ go get k8s.io/api@v${VERSION} k8s.io/apimachinery@v${VERSION} k8s.io/client-go@v${VERSION} k8s.io/mount-utils@v${VERSION}
+VERSION=<upgrading Kubernetes release version>
+go get k8s.io/api@v${VERSION} k8s.io/apimachinery@v${VERSION} k8s.io/client-go@v${VERSION} k8s.io/mount-utils@v${VERSION}
```
Comment (satoru-takeuchi) [inline docs/maintenance.md:40]: Whether `$ ` is necessary or not depends on the project. In topolvm project (not only topolvm but also other tools), we specify `$ ` to clarify whether commands should be run by root or not. So I'd like to keep this code snippet as is.

**Comment by ryanfaircloth** [comment]
Feedback is the desired behavior is to allow customers to deploy outside of test matrix without the need to bypass a protection so this one can be closed

**Pull request closed without merging**